doc/usecases/draft/copy_paste_via_icons.md

```diff
@@ -0,0 +1,40 @@
+# Use Case: Copy & Paste of Selection Via Icons
```

Comment:
The UC somehow suggest that it is not possible to do ctrl+c and then use paste icon. So maybe this UC is also not needed as it seems to only indicate a development step?